ncl/ncl_index/ncl-index-table.csv

```diff
@@ -52,3 +52,18 @@ NCL Function,Description,Python Equivalent,Notes
+`boxplot <https://www.ncl.ucar.edu/Document/Functions/Shea_util/boxplot.shtml>`__,"Creates a boxplot","``matplotlib.pyplot.boxplot()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Boxplots/index.html>`__
+`skewT_PlotData <https://www.ncl.ucar.edu/Document/Functions/Skewt_func/skewT_PlotData.shtml>`__,"Plot a sounding and (optionally) winds on Skew T, Log P charts created by skewT_BackGround","``metpy.plots.SkewT()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Skew-T/index.html>`__
+`skewT_BackGround <https://www.ncl.ucar.edu/Document/Functions/Skewt_func/skewT_BackGround.shtml>`__,"Creates a background chart for Skew T, Log P plotting","``metpy.plots.SkewT()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Skew-T/index.html>`__
+`gsn_panel <https://www.ncl.ucar.edu/Document/Graphics/Interfaces/gsn_panel.shtml>`__,"Draws multiple plots of identical size on a single frame","``matplotlib.pyplot.subplot()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Panels/index.html>`__
+`gsn_csm_pres_hgt <https://www.ncl.ucar.edu/Document/Graphics/Interfaces/gsn_csm_pres_hgt.shtml>`__,"Creates and draws a pressure/height plot","``geocat.viz.util.add_height_from_pressure_axis()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Contours/NCL_h_lat_6.html>`__
+`gsn_csm_pres_hgt_vector <https://www.ncl.ucar.edu/Document/Graphics/Interfaces/gsn_csm_pres_hgt_vector.shtml>`__,"Creates and draws a pressure/height contour plot overlaid with vectors","``geocat.comp.interp_hybrid_to_pressure()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Contours/NCL_h_lat_7.html>`__
+`gsn_csm_streamline <https://www.ncl.ucar.edu/Document/Graphics/Interfaces/gsn_csm_streamline.shtml>`__,"Creates and draws a streamline plot","``matplotlib.pyplot.streamplot()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Streamlines/NCL_stream_1.html>`__
+`gsn_csm_streamline_map <https://www.ncl.ucar.edu/Document/Graphics/Interfaces/gsn_csm_streamline_map.shtml>`__,"Creates and draws a streamline plot over a map","``matplotlib.pyplot.streamplot()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Streamlines/NCL_stream_9.html>`__
+`gsn_csm_contour_map <https://www.ncl.ucar.edu/Document/Graphics/Interfaces/gsn_csm_contour_map.shtml>`__,"Creates and draws a contour plot over a map","``matplotlib.pyplot.contour()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/MapProjections/NCL_proj_1.html>`__
+`gsn_add_polyline <https://www.ncl.ucar.edu/Document/Graphics/Interfaces/gsn_add_polyline.shtml>`__,"Attaches a polyline to the given plot","``cartopy.ax.add_geometries()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Polygons/index.html>`__
```

Comment:
`matplotlib.patches.Polygon()` should cover this as well I think.  The NCL function didn't really require / handle projection info it was more of a lines / annotations on a plot thing.

```suggestion
`gsn_add_polyline <https://www.ncl.ucar.edu/Document/Graphics/Interfaces/gsn_add_polyline.shtml>`__,"Attaches a polyline to the given plot","``matplotlib.patches.Polygon()``",`geocat-examples <https://geocat-examples.readthedocs.io/en/latest/gallery/Polygons/index.html>`__
```